spec/exhibits-solr-conf/configure_solr_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'spec_helper'
+require 'rake'
+require 'hurley'
+require 'jettywrapper'
+
+describe 'exhibits namespace rake task' do
+  before do
+    load File.expand_path('../../../lib/exhibits-solr-conf/tasks/configure_solr.rake', __FILE__)
+  end
+
+  describe 'configure_solr' do
+    it { expect { Rake::Task['exhibits:configure_solr'].invoke }.not_to raise_exception }
+
+    it 'takes argument for the target solr conf dir' do
+      target_dir = "#{File.dirname(__FILE__)}/../../tmp/solr/"
+      FileUtils.remove_dir(target_dir) if Dir.exist?(target_dir)
+      Dir.mkdir(target_dir)
+      Rake::Task['exhibits:configure_solr'].reenable
+      Rake.application.invoke_task "exhibits:configure_solr[#{target_dir}]"
+      copied_files = Dir.entries(target_dir)
+      expect(copied_files.size).to be > 5
+      expect(copied_files).to include('schema.xml', 'solrconfig.xml', 'synonyms_both_anchors.txt')
+    end
+
+    it 'Solr has correct configs and responds to select' do
+      ZIP_URL = 'https://github.com/projectblacklight/blacklight-jetty/archive/v4.10.4.zip'
+
+      Rake::Task['jetty:clean'].reenable
+      Rake.application.invoke_task 'jetty:clean'
+      Rake::Task['exhibits:configure_solr'].reenable
+      Rake.application.invoke_task 'exhibits:configure_solr'
+
+      jetty_params = Jettywrapper.load_config
+      jetty_params[:startup_wait] = 20
+      Jettywrapper.wrap(jetty_params) do
+        solr_client = Hurley::Client.new('http://127.0.0.1:8983/solr')
```

Comment:
given that we use jetty for all the spotlight testing, it seemed appropriate to use it here -- it's gotta work with blacklight jetty, and it's gotta work with Solr 4.10.4 at the moment.   It's an internal stanford specific gem.  

I have heard of Bill's simple_solr.